posthog/tasks/process_event.py

```diff
@@ -161,30 +161,18 @@ def _capture(
         except IntegrityError:
             pass
 
-
-def get_or_create_person(team_id: int, distinct_id: str) -> Tuple[Person, bool]:
```

Comment:
I guess this got removed because no code anywhere is using it?

Reply:
Yup, as per description:

```
This PR also:

Removes the unused get_or_create_person
Adds a type check before setting properties on a user
```

Reply:
ah yes :D